bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java

```diff
@@ -155,4 +172,12 @@ public CloudBigtableScanConfiguration(String projectId, String zone, String clus
   public Scan getScan() {
     return serializableScan.scan;
   }
+
+  @Override
+  public Builder toBuilder() {
```

Comment:
I was thinking about doing that, but it's not practical.  Each subclass creates a different implementation of Builder.  super.toBuilder() would not have a withScan() method in it, since eventually it would return a CloudBigtableTableConfiguration.Builder instead of a CloudBigtableScanConfiguration.Builder.... it would be worse, if CloudBigtableTableConfiguration.toBuilder() returned super.toBuilder(), because then it wouldn't even have a withTableId in it since it would return a CloudBigtableConfiguration.Builder...

This whole thing is a bit convoluted, but I don't know if there's a better solution.